site/src/components/Footer.tsx

```diff
@@ -129,12 +136,16 @@ const Socials = (
     <LinkButton
       href="https://github.com/blockprotocol/blockprotocol"
       variant="primary"
+      color="gray"
       size="small"
       sx={{
         flexShrink: 0,
         marginBottom: 2.5,
         backgroundColor: (theme) => theme.palette.gray[70],
         color: (theme) => theme.palette.gray[20],
+        ":focus-visible:after": {
+          borderColor: (theme) => theme.palette.gray[70],
+        },
```

Comment:
This is the only instance of a `{ variant: "primary", color: "gray" }` button in the codebase - I'd rather we remove these values and let it use the values in the MUI theme to avoid special casing this (also gets us a hover effect)

cc @MaggieAppleton @nonparibus I can't find this footer button in the designs, or indeed any 'primary / gray' button in the design components, so I'm unsure what the 'correct' version is here - any views? Either we should update how this appears to match the theme, or update the theme (for primary/gray button) to match how this appears.

If we update this to match the theme it will look like:

https://user-images.githubusercontent.com/37743469/151825674-4b7dd36d-f7a1-428c-ae2c-c1509198caa0.mp4

Reply:
Yeah it makes sense to not keep any bespoke styles. I tried a few options:

- Sync social icons with the github button (too dark) https://github.com/blockprotocol/blockprotocol/pull/141/commits/b6117ef0cb6d5635404afc196e49fb60b4183f12

https://user-images.githubusercontent.com/608862/151839622-d1fdd7ca-57be-4892-b333-b9491361ad05.mp4

- Sync button with social icons (text contrast is too low) https://github.com/blockprotocol/blockprotocol/pull/141/commits/f87971e98a58ba14325d32bca939598ff06700e4 + [vercel](https://blockprotocol-d43a4e264-hashintel.vercel.app/)

https://user-images.githubusercontent.com/608862/151839652-35989f66-9809-4f00-b9bf-d237f8919c25.mp4


- Dark font (maybe ok?) https://github.com/blockprotocol/blockprotocol/pull/141/commits/1620d008077fcacec6bf2b13366f77d01394634c + [vercel](https://blockprotocol-o0z2b6v8p-hashintel.vercel.app/)

https://user-images.githubusercontent.com/608862/151839679-416d2748-ec9a-4eb2-9526-bc4c1750f808.mp4

Reply:
Looks good to me but will need reflecting back in the design (where the social icons are present) - let's get this in now since it massively improves keyboard nav / accessibility, and we can update these colours as needed later.